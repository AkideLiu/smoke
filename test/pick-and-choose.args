fixtures/pick/args
fixtures/pick/in
