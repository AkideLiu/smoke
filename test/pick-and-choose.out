args
  succeeded
in
  succeeded

2 tests, 0 failures
